Dockerfile

```diff
@@ -1,31 +1,32 @@
-# --------------------------------------------------------------------------
-# This is a Dockerfile to build an Ubuntu 14.04 Docker image with
-# pymssql and FreeTDS
-#
-# Use a command like:
-#     docker build -t pymssql/pymssql .
-# --------------------------------------------------------------------------
+FROM  ubuntu:18.04
+MAINTAINER  Tyler French tylerfrench2@gmail.com (@frenchtoasters)
 
-FROM  orchardup/python:2.7
-MAINTAINER  Marc Abramowitz <marc@marc-abramowitz.com> (@msabramo)
+# Install dependices
+RUN apt-get update && apt-get install -y \
+    unixodbc \
+    unixodbc-dev \
+    unixodbc-bin \
+    libodbc1 \
+    odbcinst1debian2 \
+    tdsodbc \
+    php-odbc \
+    python3 \
+    python3-pip
```

Comment:
I think it's best practice to add `&& rm -rf /var/lib/apt/lists/*` to the end of any apt install?

Reply:
I have seen something similar all over alpine based images I will add that.